app/driveshare.html

```diff
@@ -319,9 +271,9 @@ <h4 class="modal-title">Console Logs</h4>
 			], function(logs, about, userdata, updater, process, setup) {
 				logs.init();
 				about.init();
+				process.init();
```

Comment:
since the process script is now used by the userdata script, I wanted to be sure that process is going to be bootstrapped before the userdata makes use of it